Tests/UnitTests/Core/Propagator/EigenStepperTests.cpp

```diff
@@ -505,17 +505,14 @@ BOOST_AUTO_TEST_CASE(eigen_stepper_test) {
   // Update in context of a surface
   freeParams = detail::transformBoundToFreeParameters(
       bp.referenceSurface(), tgContext, bp.parameters());
-  freeParams.segment<3>(eFreePos0) *= 2;
-  freeParams[eFreeTime] *= 2;
-  freeParams[eFreeQOverP] *= -0.5;
 
   es.update(esState, freeParams, bp.parameters(), 2 * (*bp.covariance()),
             *plane);
-  CHECK_CLOSE_OR_SMALL(es.position(esState), 2. * pos, eps, eps);
+  CHECK_CLOSE_OR_SMALL(es.position(esState), pos, eps, eps);
```

Comment:
Why are these different?

Reply:
I think what the test was doing before was scaling up the global position while leaving the local one unchanged. this fails now because we check if we are actually on surface. I changed the test now to set the original position which corresponds to the bound position